### PR TITLE
rerun tp53 module using RNA file in `/data`

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
There are some samples missing from the results of #1382.

#### What was your approach?
Commented out the else part of ifelse in the run script to force run using the `/data` folder source file. 

Background:

These 7 samples are all in pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds, which is the input for the classifier. However, they are not in pbta-gene-expression-rsem-fpkm-collapsed.stranded_classifier_scores.tsv, the results of the classifier.

However, the files in OpenPBTA-analysis/analyses/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds are these exact 7 samples short. Even though there is an ifelse to run using the /data folder when not run for subtyping, it seems that although I am reading the logic as OK, the module has been using the results file from collapse-rnaseq.

#### What GitHub issue does your pull request address?
NA

#### Which areas should receive a particularly close look?
am I reading the logic wrong? why was it using the results folder file?

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### Reproducibility Checklist

- [ ] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [ ] This analysis has been added to continuous integration.

#### Documentation Checklist

- [ ] This analysis module has a `README` and it is up to date.
- [ ] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [ ] The analytical code is documented and contains comments.

### CODE REVIEW

File: analyses/tp53_nf1_score/run_classifier.sh [hunk 1]
```diff
@@ -12,20 +12,24 @@
 set -e
 set -o pipefail
 
+# If running for subtyping, use the base histologies file
 RUN_FOR_SUBTYPING=${OPENPBTA_BASE_SUBTYPING:-0}
 
+# If we're running for figure generation, set to 1
```
Comment (jaclyn-taroni) [inline analyses/tp53_nf1_score/run_classifier.sh:18]: @sjspielman does this approach (more details on #1394) work for you?

**Comment by jharenza** [comment]
@jaclyn-taroni the missing samples seem to be back in the results file with https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1383/commits/61f9ba56e4a58b24a910da6a7aa275f968385124

```r
> missing <- c("BS_SB12W1XT",
+              "BS_FXJY0MNH",
+              "BS_HE0WJRW6",
+              "BS_D7XRFE0R",
+              "BS_KABQQA0T",
+              "BS_FN07P04C",
+              "BS_SHJA4MR0")
> 
> tp53_strand <- read_tsv("~/Documents/GitHub/OpenPBTA-analysis/analyses/tp53_nf1_score/results/pbta-gene-expression-rsem-fpkm-collapsed.stranded_classifier_scores.tsv")

── Column specification ──────────────────────────────────────────────────────────────────────────────────
cols(
  sample_id = col_character(),
  ras_score = col_double(),
  tp53_score = col_double(),
  nf1_score = col_double(),
  ras_shuffle = col_double(),
  tp53_shuffle = col_double(),
  nf1_shuffle = col_double()
)

> setdiff(missing, tp53_strand$sample_id)
character(0)
```

**Comment by jharenza** [comment]
I am not entirely sure why there are 1014 samples in the old file, but my guess is maybe this was last run when we had added those "polya+stranded" samples? But, we have the expected 977 stranded samples which are in the v21 histologies file in the output now.

File: analyses/tp53_nf1_score/run_classifier.sh (hunk 1 above)
Comment (sjspielman) [reply analyses/tp53_nf1_score/run_classifier.sh:18]: :100:

**Review by jaclyn-taroni — APPROVED** [review]
I think this is good to merge

**Pull request merged**